recipes/souporcell/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "souporcell" %}
+{% set version = "2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wheaton5/souporcell/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1404b87e44c4452b2f329abb81ce4a8383ed5b7e94f38ff2815c5bed62ec2c7f
+  patches:
+    - patches/0001-constrain-vcf-dep.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('souporcell', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  run:
+    - python
+    - pysam
+    - pystan <3
+    - pyvcf
+    - numpy
+    - pyfaidx
+    - scipy
+    - bcftools >=1.9
+    - bedtools >=2.28
+    - freebayes >=1.3
+    - htslib >=1.9
+    - hisat2
+    - minimap2 >=2.26
+    - samtools >=1.9
+    - vartrix
+
+test:
+  commands:
+    - souporcell_pipeline.py -h
```

Comment:
I think we need something a little more substantial here. E.g., I don't think that this checks that the binaries are in the expected locations.

Reply:
The only example data I see is resource intensive, so will have to manually test artifacts. For now, I included all binaries and scripts that are executable.

Reply:
I have tested the artifact on [the demo data](https://github.com/wheaton5/souporcell?tab=readme-ov-file#practicetesting-data-set-demuxlet-paper-data). All steps ran without issue, though maybe worth noting that I used the `--skip_remap SKIP_REMAP` option (with common variants), so it didn't test running `minimap2`.

But the scripts work and both binaries (souporcell, troublet) executed without issue.